### PR TITLE
rubocop 0.52.1

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Let's see how this goes.

### CODE REVIEW

File: Library/.rubocop.yml [hunk 1]
```diff
@@ -123,6 +123,10 @@ Style/Documentation:
 Style/Encoding:
   Enabled: true
 
+# disabled until we get the Metrics/LineLength down to 80.
```
Comment (alyssais) [inline Library/.rubocop.yml]: Is that something we're actively working on/making progress in?

Comment (MikeMcQuaid) [reply Library/.rubocop.yml]: Not currently.

**Comment by MikeMcQuaid** [comment]
@GauthamGoli any ideas on about the failures here?

**Comment by GauthamGoli** [comment]
@MikeMcQuaid Yeah https://github.com/bbatsov/rubocop/issues/3558
We have to make `autocorrect` public in all the cops. Just remove `private` keyword in following cops
 - `bottle_block_cop.rb`
 - `checksum_cop.rb`
 - `class_cop.rb`
 - `components_order_cop.rb`
 - `formula_desc_cop.rb`
 - `urls_cop.rb`

**Comment by MikeMcQuaid** [comment]
Thanks @GauthamGoli! Let's give it another shot.

**Comment by MikeMcQuaid** [comment]
CC @reitermarkus Can you add `rubocop-cask` support? Thanks!

**Comment by ilovezfs** [comment]
This PR caused `--display-cop-names` to become the default. So there is now a redundant command line option in both `brew audit` and `brew style`, and there is no way to suppress the display of the fairly-useless-to-users cop names.

**Comment by ilovezfs** [comment]
https://github.com/Homebrew/brew/pull/3710

**Pull request merged**